#!/usr/bin/env python
PACKAGE = "high_level_controller"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("kp_pos", double_t, 0, "Nonlinear diff drive k gain pos",    5, 1,   1000)
gen.add("kp_theta", double_t, 0, "Nonlinear diff drive k gain angle",    100, 1,   1000)

exit(gen.generate(PACKAGE, "high_level_controller", "controller_diff_drive"))
